README.md

```diff
@@ -242,6 +242,42 @@ const client = new faunadb.Client({
 })
 ```
 
+#### Termination
```

Comment:
@retwere Makes sense, I've changed the section title to "HTTP2 Session Idle Time (NodeJS only)".

> it might be good to split this discussion into two sections, one for termination...

I couldn't figure out what we can put into the "Termination" section as it only makes sense in the context of `http2SessionIdleTime` for now and is directly related to the "HTTP2 Session Idle Time" section.